server/models/Pet.js

```diff
@@ -5,9 +5,9 @@ const pool = new pg.Pool({
 })
 
 class Pet {
-  constructor({ id, name, age, vaccination_status, vaccinationStatus, adoption_story, adoptionStory, available_for_adoption, availableForAdoption, pet_type_id, petTypeId, type, img_url, imgUrl }) {
+  constructor({ id, name, petName, age, vaccination_status, vaccinationStatus, adoption_story, adoptionStory, available_for_adoption, availableForAdoption, pet_type_id, petTypeId, type, img_url, imgUrl }) {

```

Comment:
We shouldn't be taking in a different name here -- what is the goal? If it's to take a parameter from our Router, the better move would be to hand it in with a key of `name` via the router

Reply:
It's because our post method takes req.body in for all the information required for a new surrender application and pet at once and pets and surrender applications both have name keys so it is differentiating for the method.  The models for both pet and surrender application are set up to recognized keys for both.  Otherwise the post method may assign human names to pets and vice versa.  Our save surrender application method is set up to take in the pet id from the pet save method so they rely on each other

Reply:
I would argue that this should then be a concern of the controller, not the model. We don't want to over-extend the signature of the model's constructor by making it _too_ flexible. Instead, in our controller, can we use the spread operator to fill this in correctly?
```
const newPet = new Pet({...req.body, name: req.body.petName})
const newSurrender = new SurrenderApplication({...req.body, name: req.body.personName})
```

Reply:
It is also for readability and clarity on the form component since it was confusing having two name keys (one for pet and one for the owner. I hadn't considered using the spread operator in the model like that.  I will try to refactor now

Reply:
Okay!  Got the refactor working